config/config.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+AVAILABLE_DEPLOYMENTS=" deepcarbon.net
+ deepcarbon.net-staging
+ udco.tw.rpi.edu
+ dcotest.tw.rpi.edu"
+
+replace() {
+`find ../*.html -type -f -print0 | xargs -0 sed -i "s%$1%$2%g"`
+`find ../ontologies -type -f -print0 | xargs -0 sed -i "s%$1%$2%g"`
+`find ../opensearch -type -f -print0 | xargs -0 sed -i "s%$1%$2%g"`
+`find ../js -type -f -print0 | xargs -0 sed -i "s%$1%$2%g"`
+}
+
+case $1 in
+    --d=*|--deployment=*)
+    DEPLOYMENT="${1#*=}"
+
+    case ${DEPLOYMENT} in
+
+        "deepcarbon.net")
+        # TODO add URL replacement assertions (if needed)
+        echo "configured for deployment to ${DEPLOYMENT}"
+        ;;
+
+        "deepcarbon.net-staging")
+        replace "https://data.deepcarbon.net/browsers/" "https://data.deepcarbon.net/staging/browsers/"
+        replace "https://data.deepcarbon.net/s2s/" "https://data.deepcarbon.net:8444/s2s/"
+        replace "http://fuseki:3030/vivo/query" "http://deepcarbon.tw.rpi.edu:3030/VIVO/query"
```

Comment:
I would like the default case to be that fuseki is deployed using docker in a container identified to the docker internal network as 'fuseki'.  In that case the container running the PHP code would be able to access the endpoint as http://fuseki:3030 on the docker 'internal network'.

This would mean that the endpoint URL would not have to change between deployments.